docs/tutorials/ort-training.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Accelerate PyTorch training
-grand_parent: Tutorials
-parent: Training
+parent: Tutorials
```

Comment:
currently this page shows up first under Tutorials. Feel like it should be a subpage of the "Accelerate PyTorch" page. or at least it should show up second

Reply:
This is a good catch! This round I just moved things under the tutorials nav with the rework. I plan to do a lot with tutorials on future updates. I created the sub nav for "Accelerate PyTorch" and it looks much better.